src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -208,12 +128,24 @@ def _get_stub_and_run_id(self) -> Tuple[GrpcDriverStub, int]:
                 fab_version=res.run.fab_version,
             )
 
-        return self.stub, self._run.run_id
+    @property
+    def run(self) -> Run:
+        """Run information."""
+        self._init_run()
+        return Run(**vars(self._run))
+
+    @property
+    def stub(self) -> DriverStub:
```

Comment:
This should be private (`_stub`). We do not want to expose `stub` as part of the public API of `GrpcDriver`, even though we use it in the `run_serverapp` module (for now, we will move `CreateRun` out of the Driver API soon).

Reply:
That makes a lot of sense. Thanks for bringing it up. I've updated it.